operator/controllers/kyma_controller.go

```diff
@@ -253,7 +272,14 @@ func (r *KymaReconciler) HandleProcessingState(ctx context.Context, kyma *v1alph
 		return r.UpdateStatusWithEventFromErr(ctx, kyma, v1alpha1.StateError,
 			fmt.Errorf("error while syncing conditions during deleting non exists modules: %w", err))
 	}
-
+	statusUpdateRequiredFromSKRWebhookSync := false
+	if r.EnableKcpWatcher {
```

Comment:
I have reused the syncCtx in the deleting part. In the processing logic there is no syncCtx to reuse.

Reply:
the syncCtx is created even before going into processing. I think this might need a bigger refactoring since some parts that require the sync context need the kyma sync to be enabled while others do not.

Reply:
Sure, let's do it in [this](https://github.com/kyma-project/lifecycle-manager/issues/296) follow-up issue.